sass/components/rtl_support/_badges.scss

```diff
@@ -0,0 +1,17 @@
+// RTL Badges
+.rtl {
+	span.badge {
+		margin-left: initial;
+		margin-right: 14px;
+		float: left;
+
+		&.new:after {
+			content: " جدید";
```

Comment:
@NonameSLdev I'm not sure if JS is the most optimized solution for this case. We're talking about static content here, therefore it won't worth it to bother client's memory load with JS.
Data-attribute would be the best option as soon as it gets well-supported by all browsers, till then we could either load each data-attribute into the tag with JS or let the developer set their own "phrase" inside a span tag.